anx/provider/loadbalancer/address/address.go

```diff
@@ -54,9 +55,9 @@ func NewWithPrefixes(ctx context.Context, apiClient api.API, legacyClient client
 }
 
 // NewWithPrefixAutodiscovery creates a new Manager instance doing Prefix autodiscovery
-func NewWithPrefixAutodiscovery(ctx context.Context, apiClient api.API, legacyClient client.Client, prefixTag string) Manager {
-	m := newMgr(ctx, apiClient, legacyClient)
-	m.prefixAutodiscover = &prefixTag
+func NewWithPrefixAutodiscovery(ctx context.Context, apiClient api.API, legacyClient client.Client, clusterName string) Manager {
+	m := newMgr(ctx, apiClient, legacyClient, clusterName)
+	m.prefixAutodiscover = pointer.String(fmt.Sprintf("kubernetes-lb-prefix-%s", clusterName))
```

Comment:
what do you think about having `m.autodiscoverName` containing the cluster name if autoconfig is enabled, instead of having two members for that?

Reply:
Only using `m.autoDiscoverName` now